### PR TITLE
Update committer page

### PR DESCRIPTION
**Pull request by jerryshao**
Adding me to the committers page.

### CODE REVIEW

**Comment by jerryshao** [comment]
@srowen would you please help to review this changes.

File: committers.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ navigation:
 |Burak Yavuz|Databricks|
 |Matei Zaharia|Databricks, Stanford|
 |Shixiong Zhu|Databricks|
+|Saisai Shao|Hortonworks|
```
Comment (HyukjinKwon) [inline committers.md]: @jerryshao, it might be not a big deal but I remember I placed my name in alphabetical order of last name ..

Comment (jerryshao) [reply committers.md]: Ohh, I see, I will change it.

**Comment by jerryshao** [comment]
How about now? @HyukjinKwon .

**Review by HyukjinKwon — APPROVED** [review]

**Comment by srowen** [comment]
Looks good to me. I have a small problem -- the usual way I use to merge commits won't work here because the source branch has the same name as my `asf-site` branch. (].Usually I `git fetch` the PR branch and just rebase/push it to the ASF git `asf-site`.

I can easily apply this as a patch and push, but does anyone else know the right-er way to do it? I might learn an easier way.

**Comment by jerryshao** [comment]
@srowen @HyukjinKwon please guide me how should I do, should I directly push the changes to ASF spark-website repo, or what else should I do?

**Comment by srowen** [comment]
Yeah you should be able to push too. If your remotes are like ...

```
apache	https://git-wip-us.apache.org/repos/asf/spark-website.git (fetch)
apache	https://git-wip-us.apache.org/repos/asf/spark-website.git (push)
...
upstream	https://github.com/apache/spark-website.git (fetch)
upstream	https://github.com/apache/spark-website.git (push)
```

... then you should be able to first update your `asf-site` branch, and then just push to the `apache` repo `asf-site` branch, as in `git push apache asf-site`.

**Comment by jerryshao** [comment]
I see, you mean that I can directly push to ASF spark-website repo, unlike Spark which uses a script.

**Comment by srowen** [comment]
Yeah, you have to do it manually here (so I have to be careful when I do it) but changes are so rare that it's generally no big deal.

**Comment by jerryshao** [comment]
OK, I just pushed a new commit, and this could be closed.

**Pull request closed without merging**